src/nccl_ofi_rdma.c

```diff
@@ -4939,20 +4905,7 @@ static int blocked_send_close(nccl_net_ofi_send_comm_t *send_comm)
 		return -EINVAL;
 	}
 
-	// TODO: We might want to use READ_ONCE to read variable `connected'
-	int ret = 0;
-	while (!s_comm->connected) {
-		__compiler_barrier();
-		/* Progress our engine to get completions. If the
-		 * connect response message has arrived, the
-		 * connection establishment will be finalized. */
-		ret = ofi_process_cq(ep);
-		if (OFI_UNLIKELY(ret != 0)) {
-			return ret;
-		}
-	}
-
-	ret = send_close(s_comm);
+	int ret = send_close(s_comm);
```

Comment:
why not just `return send_close(s_comm);`?

Reply:
We actually don't need `blocked_send_close` at all anymore.

Reply:
Removed.